docs/cow-protocol/reference/core/tokens.md

```diff
@@ -30,3 +30,11 @@ There is an explicit bad token list, containing tokens that have been identified
 One such example is tokens that take a fee on transfer.
 The bad token list is not exhaustive and is supplemented by automated bad token detection, which may be flaky, (e.g. if a token exhibits rounding issues on transfers) depending on certain conditions.
 In any case, if a token is on the bad token list, it will result in an `UnsupportedToken` error when attempting to retrieve a quote or place an order.
+
+## New ERC-20 tokens
+
+Have you created a new `ERC-20` and want to make it seemlessly tradeable on CoW Protocol?
+
+- Bootstrap a Uni v2 (or some other well-known AMM) pool with the token (and it's minimum viable liquidity)
+- Potentially reach out to solvers (contact us via Discord) to have any optimal execution paths added to their solvers (such as a path through the newly created pool)
```

Comment:
Seems to over promise a bit.
Maybe "reach out to solvers to let them know about the best liquidity source and how they need to handle the token"